imageoptim/ImageOptimController.m

```diff
@@ -248,18 +250,36 @@ - (void)loadCreditsHTML:(id)_unused {
         documentAttributes:nil];
 
 
-    NSTextView *creditsRef = credits;
     dispatch_async(dispatch_get_main_queue(), ^() {
-      [creditsRef setEditable:YES];
-      [creditsRef insertText:tmpStr replacementRange:NSMakeRange(0, 0)];
-      [creditsRef setEditable:NO];
+      [self->credits setEditable:YES];
+      [self->credits insertText:tmpStr replacementRange:NSMakeRange(0, 0)];
+      [self->credits setEditable:NO];
+      [self adaptCreditsAppearance];
     });
 }
 
+- (BOOL)isDarkMode {
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101400
+    if (@available(macOS 10.14, *)) {
+        NSAppearanceName bestAppearance = [credits.effectiveAppearance bestMatchFromAppearancesWithNames:@[NSAppearanceNameAqua, NSAppearanceNameDarkAqua]];
+        return [bestAppearance isEqualToString: NSAppearanceNameDarkAqua];
+    }
+#endif
+    return false;
+}
+
+- (void)adaptCreditsAppearance {
+    credits.textColor = [self isDarkMode] ? [NSColor whiteColor] : [NSColor blackColor];
```

Comment:
Why not just [`[NSColor textColor]`](https://developer.apple.com/documentation/appkit/nscolor/1527025-textcolor)?